samples/client/echo_api/python-disallowAdditionalPropertiesIfNotPresent/openapi_client/api_client.py

```diff
@@ -366,7 +371,11 @@ def sanitize_for_serialization(self, obj):
             # and attributes which value is not None.
             # Convert attribute name to json key in
             # model definition for request.
-            obj_dict = obj.to_dict()
+            if hasattr(obj, 'to_dict') and callable(getattr(obj, 'to_dict')):
+
```

Comment:
can you please remove this extra line break?

Reply:
> can you please remove this extra blank line?

done